src/index.ts

```diff
@@ -290,4 +291,27 @@ Please refer to the Monika documentations on how to how to configure notificatio
   }
 }
 
+const em = getEventEmitter()
+
+// Subscribe FirstEvent
+em.addListener('TERMINATE_EVENT', function (data) {
+  log.info('Monika Event: ' + data)
+})
+
+/**
+ * Show Exit Message
+ */
+process.on('SIGINT', () => {
+  if (!process.env.DISABLE_EXIT_MESSAGE) {
+    log.info('\nThank you for using Monika!')
```

Comment:
Please remove the \n at the beginning. It looks weird in the console output.
![Uploading Screenshot 2021-06-28 at 10.21.50.png…]()

Reply:
Fixed